x-pack/plugins/ml/public/application/settings/calendars/list/calendars_list.js

```diff
@@ -104,6 +106,8 @@ export class CalendarsListUI extends Component {
     const { canCreateCalendar, canDeleteCalendar } = this.props;
     let destroyModal = '';
 
+    const helpLink = getDocLinks().links.ml.guide;
```

Comment:
Is it worth linking to the Calendars [page](https://www.elastic.co/guide/en/machine-learning/master/ml-calendars.html) here rather than the top level of the ML guide?